wiki/cpp/DEVELOPER_GUIDE.md

```diff
@@ -368,7 +368,7 @@ This can be ensured by introducing interstream dependencies with CUDA events and
 ```cpp
 void cumlAlgo(const ML::cumlHandle& handle, ...)
```

Comment:
this should be `raft::handle_t`?

Reply:
same thing applies to other places in this doc too.

Reply:
Yes, I was thinking of leaving doc changes for the very last stage of this PR if that's acceptable

Reply:
sure Divye. I was mostly putting this out so that even us, reviewers won't miss this.